.travis.yml

```diff
@@ -8,7 +8,8 @@ addons:
 
 cache:
     directories:
-        - $HOME/.composer/cache/files
+        - $HOME/.cache/pip
+        - $HOME/.composer/cache
```

Comment:
https://github.com/travis-ci/travis-ci/issues/4579#issuecomment-138548147

> The repo subfolder in the cache contains metadata downloaded from Packagist (and other repos you may have registered in the project), and these would probably have changed for each build (the main files giving access to others are updated every few minutes because of the frequency of changes in packages). I'm not sure the time won thanks to the caching of some of the metadata files (the ones for packages not updated since your previous Travis build) would actually outweight the time spent storing the Travis cache again for each build.